docs/src/main/sphinx/admin/fault-tolerant-execution.md

```diff
@@ -46,6 +46,7 @@ connector. The following connectors support fault-tolerant execution:
 - {ref}`Iceberg connector <iceberg-fte-support>`
 - {ref}`MongoDB connector <mongodb-fte-support>`
 - {ref}`MySQL connector <mysql-fte-support>`
+- {ref}`MariaDB connector <mariadb-fte-support>`
```

Comment:
Move to before MongoDB. This list is sorted by alphabetical order.

Reply:
Done.